notebooks/wav2lip/wav2lip.ipynb

```diff
@@ -0,0 +1,326 @@
+{
```

Comment:
<p>can you please show reference audio too? it is also better to include logs for this cells that users can see expected result (or maybe provide some static example of expected result in markdown if you think that using videos makes notebook size too large)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2417/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>Output is added, they are not too large: 2643KB is size of notebook in total</p>